api/v1/helpers/verbs/findUtils.js

```diff
@@ -196,6 +198,12 @@ function toSequelizeWhere(filter, props) {
       else if (props.tagFilterName && key === props.tagFilterName) {
         const tagArr = filter[key];
         Object.assign(where, toWhereClause(tagArr, props));
+      } else if (props.timePeriodFilters &&
```

Comment:
Does doing this in an else-if mean we can't filter on both createdAt and tags?

Reply:
This is inside a for loop, so we are creating a where clause for each key at a time. I should probably add a test in one of the model to specifically test this.

Reply:
ah ok.